docs/tutorial_bracket_strategy.md

```diff
@@ -174,50 +178,50 @@ Now, you should just follow the link and sign & execute the transaction with the
 For executing, just press the button _confirm_.
 
 If you are executing the second transaction with a lower gasPrice than the first one, then you can sign it immediately.
-Otherwise, you should wait until the first one is mined. This works as the gas price usually determines the mining order of the transaction.
+Otherwise, you should wait until the first one is mined. This works in most cases, as the gas price usually determines the mining order of the transaction.
```

Comment:
Change reminder: it's not possible to execute both transactions at the same time from the web interface.